output/gigpress_sidebar.php

```diff
@@ -164,6 +166,25 @@ function form( $instance ) {
 			</select>
 		</p>
 
+		<p>
+			<label for="<?php echo esc_attr( $this->get_field_id( 'related' ) ); ?>">
+				<?php _e( 'Only display shows related to this post', 'gigpress' ); ?>
```

Comment:
```suggestion
				<?php esc_html_e( 'Only display shows related to this post', 'gigpress' ); ?>
```
Should translation strings like this be escaped?